src/device/permission/permission.ios.ts

```diff
@@ -14,14 +14,25 @@ class PermissionIOSClass extends NativeEventEmitterComponent<PermissionEvents, a
   }
   async requestPermission(permission: Parameters<IPermission['requestPermission']>['0']): Promise<PermissionResult> {
     // const requestTexts = options?.requestTexts || {};
-    const status = this.ios?.getAuthorizationStatus?.(Permissions.IOS[permission]);
+    let _permission = permission;
+    if (permission === Permissions.camera) {
+      _permission = "CAMERA";
+    } else if (permission === Permissions.location || permission === Permissions.location.approximate || permission === Permissions.location.precise) {
+      _permission = "LOCATION";
+    } else if (permission === Permissions.storage || permission === Permissions.storage.readImageAndVideo || permission === Permissions.storage.readAudio) {
+      //@ts-ignore
+      _permission = "GALLERY";
+    }
+    //@ts-ignore
```

Comment:
Why we are ignoring the typescript error here 🤔

Reply:
Fixed the ts-ignore issues.